source/draw/gpu/opengl/shaders/unpackRgbaToMonochrome.frag

```diff
@@ -0,0 +1,10 @@
+#version 300 es
+precision mediump float;
+uniform sampler2D texture0;
+uniform int targetWidth;
+in highp vec4 fragmentTextureCoordinate;
+out float outColor;
+void main() {
+  int pixelIndex = int(float(targetWidth) * fragmentTextureCoordinate.z) % 4;
+  outColor = texture(texture0, fragmentTextureCoordinate.xy)[pixelIndex];
```

Comment:
So... if you fix it now, the next time you modify that same file, Atom will keep the tab tabs, because they were already there. But, there really should be a plugin.